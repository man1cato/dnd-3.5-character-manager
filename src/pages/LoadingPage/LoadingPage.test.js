import React from 'react'
import { render } from '@testing-library/react'

import LoadingPage from './LoadingPage'


test('should render loading page correctly', () => {
    const { container } = render(<LoadingPage />)
    expect(container.firstChild).toMatchSnapshot()
})