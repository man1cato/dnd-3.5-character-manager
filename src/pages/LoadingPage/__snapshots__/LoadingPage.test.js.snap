// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading page correctly 1`] = `
<div
  class="loader"
>
  <img
    class="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
